wordpress-mobile/gutenberg-mobile
# Display content information metrics

Fixes #506

 - WordPress-iOS PR: https://github.com/wordpress-mobile/WordPress-iOS/pull/14305
 - WordPress-Android PR: https://github.com/wordpress-mobile/WordPress-Android/pull/12186
   - Follow up Android PR: https://github.com/wordpress-mobile/gutenberg-mobile/pull/2383
   - Follow up WP-Android PR: https://github.com/wordpress-mobile/WordPress-Android/pull/12191
 - Gutenberg PR: https://github.com/WordPress/gutenberg/pull/23115

This PR changes the bridge method to display the content information metrics about the content being edited:

| iOS | Android |
|-----|-----------|
| <img src="https://user-images.githubusercontent.com/651601/84710175-9bdc1d00-af5b-11ea-8a68-404c836ba10c.png" width=300> | <img src="https://user-images.githubusercontent.com/651601/84710184-a1396780-af5b-11ea-842e-3f749d6b0ceb.png" width=300> |

To test:
 - Start the demo app
 - iOS: Press the ... button on the top right, For Android you need to use the main app.
 - Check if the content metrics show on the top of the action sheet.

Notes:
 I considered implementing this in the main apps only, but in the end, I thought it was better to use a single source of the metrics for all three implementations: web, iOS, Android. 
If we implemented in the main apps we will need to replicate the code in three places, and the only marginal advantage could be some performance, wich I don't think it's worth the maintenance costs.

PR submission checklist:
